src/ExtInterface/ReefMod/ReefModDomain.jl

```diff
@@ -338,40 +350,48 @@ function _cyclone_mortality_scens(dom_dataset, spatial_data, site_ids, timeframe
         cyclone_mortality_scens[species=At(m)] .= cm_scens_massives
     end
 
-    # Set branchings deeper than 5 mortality rates
-    mask_d5::BitVector = spatial_data.Y_COORD .<= -5
-    if sum(mask_d5) > 0
+    # Set mortality rates for branching corals at <= 5m depth
+    below_5::BitVector = spatial_data.Y_COORD .<= -5
+    if sum(below_5) > 0
         mr_bd5::Vector{Float64} = cyclone_mr[:branching_deeper_than_5]
-        cm_scens_bd5::Array{Float64} = mr_bd5[cyclone_scens[location=mask_d5]].data
+        cm_scens_bd5::Array{Float64} = mr_bd5[cyclone_scens[location=below_5]].data
         for b in species[branchings]
-            cyclone_mortality_scens[location=(mask_d5), species=At(b)] .= cm_scens_bd5
+            cyclone_mortality_scens[locations=(below_5), species=At(b)] .= cm_scens_bd5
         end
     end
 
-    # Set branchings shallower than 5 mortality rates
-    mask_s5::BitVector = spatial_data.Y_COORD .> -5
-    if sum(mask_s5) > 0
+    # Set mortality rates for branching corals at > 5m depth
+    above_5::BitVector = spatial_data.Y_COORD .> -5
+    if sum(above_5) > 0
         mr_bs5::Vector{Float64} = cyclone_mr[:branching_shallower_than_5]
-        cm_scens_bs5::Array{Float64} = mr_bs5[cyclone_scens[location=(mask_s5)]].data
+        cm_scens_bs5::Array{Float64} = mr_bs5[cyclone_scens[location=(above_5)]].data
         for b in species[branchings]
-            cyclone_mortality_scens[locations=(mask_s5), species=At(b)] .= cm_scens_bs5
+            cyclone_mortality_scens[locations=(above_5), species=At(b)] .= cm_scens_bs5
         end
     end
```

Comment:
@Zapiano is there a reason why are we using latitude (`spatial_data.Y_COORD`) here?

Reply:
It should be:

`spatial_data.depth_med .>= 5` for the "deeper than 5m" case, and
`spatial_data.depth_med .< 5` for the "shallower than 5m" case

Reply:
@ConnectedSystems hmm, you're right, but on this line https://github.com/open-AIMS/ADRIA.jl/blob/5e807ded90b9f68737b548b3e30dcc054118d306/src/ExtInterface/ReefMod/ReefModDomain.jl#L126 we are setting all `depth_med` to 6.0 - reason why I thought of using the `Y_COORD`.

Reply:
That's because ReefMod assumes all locations are 7m deep. And yes, I realise the implications for cyclones.

(Yes, I know it says 6 here, it's corrected elsewhere)

Reply:
@ConnectedSystems I've replaced `Y_COORD` by `depth_med`